src/pages/saksoversikt/søkefelt/Søkefelt.tsx

```diff
@@ -1,16 +1,13 @@
 import { Input } from 'nav-frontend-skjema';
 import React from 'react';
-import { useHistory } from 'react-router-dom';
 
 import * as personSlice from '~features/person/person.slice';
 import * as sakSlice from '~features/saksoversikt/sak.slice';
-import * as Routes from '~lib/routes';
 import { useAppDispatch } from '~redux/Store';
 
-const Søkefelt = () => {
+const Søkefelt = (props: { onSuccess: (id: string) => void }) => {
```

Comment:
Det er ikke helt åpenbart hva slags `id` man egentlig får inn til `onSuccess` (siden det både søkes opp person og søknad).
Forslag
```suggestion
const Søkefelt = (props: { onSakFetchSuccess: (id: string) => void }) => {
```